types/i18next-express-middleware/index.d.ts

```diff
@@ -2,6 +2,7 @@
 // Project: http://i18next.com/
 // Definitions by: Cyril Schumacher <https://github.com/cyrilschumacher>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
```

Comment:
Tests failed because you removed this line from i18next-node-fs-backend (it does not have `"i18next": ["i18next/v2"]` mappings in tsconfig). To make i18next-node-fs-backend use v2 definitions add the same lines as you added to 18next-express-middleware tsconfig.json.